app/src/main/java/co/nimblehq/compose/crypto/extension/BoxShadow.kt

```diff
@@ -0,0 +1,53 @@
+package co.nimblehq.compose.crypto.extension
+
+import android.graphics.BlurMaskFilter
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+fun Modifier.boxShadow(color: Color = Color.Black,
+                               borderRadius: Dp = 0.dp,
+                               blurRadius: Dp = 0.dp,
+                               offsetY: Dp = 0.dp,
+                               offsetX: Dp = 0.dp,
+                               spread: Float = 0f,
+                               modifier: Modifier = Modifier,
```

Comment:
Seems your formatting style is different from the style we are currently using. Could you please reformat the update with default style in Android Studio? 😅

Reply:
@hoangnguyen92dn Ok will reformat the style

It will be very grateful if you ensure that below formatting style is the recommended style for that project :)

```
fun Modifier.boxShadow(
                              color: Color = Color.Black,
                               borderRadius: Dp = 0.dp,
                               blurRadius: Dp = 0.dp,
                               offsetY: Dp = 0.dp,
                               offsetX: Dp = 0.dp,
                               spread: Float = 0f,
                               modifier: Modifier = Modifier
)
```

Reply:
@wasim15185 Ops. It looks weird and different from the formatting style we are using 😄